README.md

```diff
@@ -58,10 +62,23 @@ conda activate bsoid_v2
 
 You should now see (bsoid_v2) $yourusername@yourmachine ~ %
 
-#### Step 2: Run the app!
+#### Step 2: Run the pipeline through the CLI!
+##### See run_streamlit_cli.py and run_streamlit_cli_predict.py for explanations of the environmental variables that need to be set here and default settings where they exist.
```

Comment:
wait im confused. the user has to go into those python scripts to look at the documentation? would it be simple to put that in the readme? if not, can you at least link to the py files?

Reply:
added to readme